TESTS/integration/fs-threaded/main.cpp

```diff
@@ -160,6 +160,7 @@ static control_t file_4_threads(const size_t call_count)
 utest::v1::status_t greentea_setup(const size_t number_of_cases)
 {
     GREENTEA_SETUP(10 * 60, "default_auto");
+    mbed_trace_init();
```

Comment:
Is this necessary since #11695? Does it conflict?

(And if we were to set other settings here, would that conflict with #11695? What's the call order?)

Reply:
@kjbracey-arm Ah, I was unaware of #11695! I believe then that it'd be safe to omit the call to `mbed_trace_init()` in the test setup functions.